src/App.jsx

```diff
@@ -118,6 +119,7 @@ function App() {
               <div className="relative grow overflow-hidden">
                 <Routes>
                   <Route path="/" element={<FrontPage />} />
+                  <Route path="/route/:id/points" element={<RoutePoints />} />
```

Comment:
Should routes consistently start with `/` – or not start with `/`?

Reply:
I am currently rewriting this..... so perhaps next pr?